k8-operator/scripts/generate-helm.sh

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env bash
+set -euo pipefail
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+PROJECT_ROOT=$(cd "${SCRIPT_DIR}/.." && pwd)
+HELM_DIR="${PROJECT_ROOT}/../helm-charts/secrets-operator"
+LOCALBIN="${PROJECT_ROOT}/bin"
+KUSTOMIZE="${LOCALBIN}/kustomize"
+HELMIFY="${LOCALBIN}/helmify"
+
+
+cd "${PROJECT_ROOT}"
+# first run the regular helm target to generate base templates
+"${KUSTOMIZE}" build config/default | "${HELMIFY}" "${HELM_DIR}"
```

Comment:
wait is this redundant with the `manifests kustomize helmify` that we already do before calling generate-helm?